.github/workflows/electron-linux.yml

```diff
@@ -0,0 +1,47 @@
+name: Electron Rebuild CI - Linux (x32 & x64)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        electron:
+          - 5
+          - 6
+          - 7
+          - 8
+        include:
+          # includes a new variable for each electron (essentially a switch statement) 
+          - electron: 5
+            nodeVersion: 70
+          - electron: 6
+            nodeVersion: 73
+          - electron: 7
+            nodeVersion: 75
+          - electron: 8
+            nodeVersion: 76
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        os: ${{ matrix.os }}
+    - name: Install packages
+      run: npm install --unsafe-perm
+    - name: Install electron-rebuild
+      run: npm install --save-dev electron-rebuild@latest
+    - name: Run on ${{ matrix.os }} - Electron v${{ matrix.electron }} (Node ${{ matrix.nodeVersion }})
+      run: ./node_modules/.bin/electron-rebuild -v=${{ matrix.electron }} -w node-sass
```

Comment:
If you use `npx` like `npx electron-rebuild -v=${{ matrix.electron }} -w node-sass` you can skip the previous global install.
Don't feel stronly though, since it looks like this works

Reply:
Just to make sure I understood you correctly, I removed the below step and ran your suggestion
```
npm install --unsafe-perm
```


If so, it produced this error. It needs to `install` to create the `binding.node` it's going to clone

```console err
internal/modules/cjs/loader.js:979
  throw err;
  ^

Error: Cannot find module 'nan'
Require stack:
- C:\Users\Glenn\Documents\GitHub\node-sass\[eval]
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:976:15)
    at Function.Module._load (internal/modules/cjs/loader.js:859:27)
    at Module.require (internal/modules/cjs/loader.js:1036:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:128:20)
    at Object.runInThisContext (vm.js:319:38)
    at Object.<anonymous> ([eval]-wrapper:10:26)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at evalScript (internal/process/execution.js:94:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'C:\\Users\\Glenn\\Documents\\GitHub\\node-sass\\[eval]' ]
}
gyp: Call to 'node -e "require('nan')"' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (C:\Users\Glenn\Documents\GitHub\node-sass\node_modules\electron-rebuild\node_modules\node-gyp\lib\configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:316:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Windows_NT 10.0.18363
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Glenn\\Documents\\GitHub\\node-sass\\node_modules\\electron-rebuild\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--target=8.0.0" "--arch=x64" "--dist-url=https://www.electronjs.org/headers" "--build-from-source"
gyp ERR! cwd C:\Users\Glenn\Documents\GitHub\node-sass
gyp ERR! node -v v13.10.1
gyp ERR! node-gyp -v v6.1.0
gyp ERR! not ok
```

Reply:
Got what you meant 🤦‍♂️. Switched it, looks cleaner.